### DISCUSSION TITLE
Solo due cifre decimali

### DISCUSSION THREAD

**Discussion by mriitan** [indomus/forum#1532] [HUB: Home Assistant]
qualcuno può aiutarmi? che cosa devo aggiungere per riuscire a visualizzare solo due cifre decimali? grazie
![image](https://user-images.githubusercontent.com/98173792/155165711-8c093ebc-532f-48aa-be4b-8173c3d16466.png)

**Comment by 50494554524F** [comment]
creati dei sensori derivati da quelli che hai ed usa il filtro "round(2)" prima di restituire il valore

**Comment by vinny74man** [comment]
Ma i sensori dai quali prendi i dati non hanno attributo precision?
Io per dei sensori modbus ho quell'attributo e mettendo `precision: 2` visualizzo solo due cifre decimali in qualsiasi card

**Reply by mriitan** [reply]
no, non lo accetta
